pkg/types/netid.go

```diff
@@ -112,6 +112,12 @@ func (id NetID) IDBits() uint {
 	panic(unmatchedNetID)
 }
 
+// Copy stores a copy of id in x and returns it.
+func (id NetID) Copy(x *NetID) *NetID {
```

Comment:
Aren't we doing something wrong when we need things like these and develop our stack in Go?

Reply:
Almost all types in `types` have this method already, so I'm just being consistent with the rest of the package here.